Documentation for Augmented graphs and GNI

aditya0by0 opened the pull request:
fixes #29

aditya0by0 committed:
add doc to static gni class

aditya0by0 committed:
add aug graph readme

aditya0by0 committed:
doc for dynamic gni

aditya0by0 committed:
refine readme for gni

aditya0by0 committed:
commands on same line for compactness

Updated the README to use a more concise format for model configuration options and hyperparameters.

aditya0by0 committed:
readme - minor clarification

sfluegel05 commented:
Thanks for adding this. Could you add one or two sentences each for (a) what is an augmented GNN? (b) What do the three pooling variants do? (c) And what is the difference between GAT and GATv2 (with a reference to the paper). In addition, it would be nice to have an image that shows how a molecule is represented in the augmented GNN

sfluegel05 commented:
Thanks for adding this. Could you add one or two sentences each for (a) what is an augmented GNN? (b) What do the three pooling variants do? (c) And what is the difference between GAT and GATv2 (with a reference to the paper). In addition, it would be nice to have an image that shows how a molecule is represented in the augmented GNN

aditya0by0 committed:
Enhance README with augmented graphs explanation

Expanded the section on augmented graphs to explain the use of artificial nodes representing functional groups. Added details on connection schemes and provided commands for model and data configuration.

aditya0by0 committed:
Fix typos and improve clarity in README.md

sfluegel05 reviewed (approved):
This looks good to me. Thanks!
